components/elm/src/biogeochem/DUSTMod.F90

```diff
@@ -238,34 +237,47 @@ subroutine DustEmission (bounds, &
     real(r8) :: sumwt(bounds%begl:bounds%endl) ! sum of weights
     logical  :: found                          ! temporary for error check
     integer  :: index
-    !
+    !    
     ! constants
     !
-    real(r8), parameter :: cst_slt = 2.61_r8           ! [frc] Saltation constant
+    !real(r8), parameter :: cst_slt = 2.61_r8           ! [frc] Saltation constant, variable no longer needed -YF
     real(r8), parameter :: flx_mss_fdg_fct = 5.0e-4_r8 ! [frc] Empir. mass flx tuning eflx_lh_vegt
     real(r8), parameter :: vai_mbl_thr = 0.3_r8        ! [m2 m-2] VAI threshold quenching dust mobilization
-    !------------------------------------------------------------------------
+    real(r8), parameter :: Cd0 = 4.4e-5_r8             ! [dimless] proportionality constant 
```

Comment:
Are those parameters brought to the name list? Tunable parameters are better placed in the namelist.